mysql/test/finagle_clojure/mysql/integration_test.clj

```diff
@@ -0,0 +1,173 @@
+(ns finagle-clojure.mysql.integration-test
+  (:import (com.twitter.util Future)
+           (java.sql Date Timestamp)
+           (java.util TimeZone))
+  (:require [midje.sweet :refer :all]
+            [finagle-clojure.mysql.client :refer :all]
+            [finagle-clojure.futures :as f]
+            [finagle-clojure.scala :as scala]))
+
+(System/setProperty "java.net.preferIPv4Stack" "true")
+(TimeZone/setDefault (TimeZone/getTimeZone "UTC"))
+
+(defn mapfn [^Future fut func]
+  (f/map* fut (scala/Function* func)))
+
+(facts "integrating the MySQL client"
```

Comment:
tag this with some midje metadata so it can be easily filtered in environments where mysql isn't available.